tests/dummy/app/pods/checkbox/template.hbs

```diff
@@ -1,172 +1,154 @@
-<div class="section">
-  <div class="title">frost-checkbox</div>
-  <div class="example">
-    <div class="title">Basic</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
-{{frost-checkbox id='basic'}}
-```"
-}}
+<div class="dummy-notification-container">
+{{#each notifications as |notification|}}
+  {{notification-message notification=notification}}
+{{/each}}
+</div>
+
+<div class='dummy-body text-input field'>
+  <div class='section'>
+
+    <div class='example'>
+      <div class='title'>Basic</div>
+      <div class='demo'>
+        {{frost-checkbox}}
+      </div>
+      <div class='snippet'>
+        {{format-markdown
+"```handlebars
+{{frost-checkbox}}
+```"}}
+      </div>
     </div>
-    <div class="demo">
-      {{frost-checkbox id="basic"}}
+
+   <div class='example'>
+      <div class='title'>Size</div>
+      <div class='demo'>
+        {{frost-checkbox size='large'}}
+      </div>
+      <div class='snippet'>
+        {{format-markdown
+"```handlebars
+{{frost-checkbox
+  size='large'
+}}
+```"}}
+      </div>
     </div>
-  </div>
 
-  <div class="example">
-    <div class="title">Label</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
+    <div class='example'>
+      <div class='title'>Label</div>
+      <div class='demo'>
+        {{frost-checkbox label='label'}}
+      </div>
+      <div class='snippet'>
+        {{format-markdown
+"```handlebars
 {{frost-checkbox
-  id='label'
   label='label'
 }}
-```"
-      }}
-    </div>
-    <div class="demo">
-      {{frost-checkbox
-        id='label'
-        label='label'
-      }}
+```"}}
+      </div>
     </div>
-  </div>
 
-  <div class="example">
-    <div class="title">Autofocus</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
+    <div class='example'>
+      <div class='title'>Autofocus</div>
+      <div class='demo'>
+        {{frost-checkbox autofocus='true'}}
+      </div>
+      <div class='snippet'>
+        {{format-markdown
+"```handlebars
 {{frost-checkbox
-  id='autofocus'
-  autofocus=true
-  label='autofocus'
+  autofocus='true'
 }}
-```"
-      }}
-    </div>
-    <div class="demo">
-      {{frost-checkbox
-        id='autofocus'
-        autofocus=true
-        label='autofocus'
-      }}
+```"}}
+      </div>
     </div>
-  </div>
 
-  <div class="example">
-    <div class="title">Checked</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
+    <div class='example'>
+      <div class='title'>Checked</div>
+      <div class='demo'>
+        {{frost-checkbox checked='true'}}
```

Comment:
It is a boolean, added '' to make it standout, but I guess that is not correct - the readme has it as boolean.